apps/website/src/pages/docs/expandableblock.mdx

```diff
@@ -70,9 +75,9 @@ An expandable block can be disabled by using the `disabled` prop.
   <AllExamples.ExpandableBlockDisabledExample client:load />
 </LiveExample>
 
-## Usage
+## Guidelines
```

Comment:
We use *"Usage"* in other places on the website. So should we do the same here? Why did you change to *"Guidelines"*?

Reply:
Because "Usage" doesn't make sense here. Usage is more about *how* to use it (like the section near the top of this file). I'm quite unhappy with how most pages on the website are structured.